src/lv_core/lv_indev.c

```diff
@@ -310,6 +323,14 @@ lv_task_t * lv_indev_get_read_task(lv_disp_t * indev)
     return indev->refr_task;
 }
 
+lv_obj_t * lv_indev_get_obj_focused() {
```

Comment:
Please add a comment above this function and explicitly add `(void)` parameters.

Reply:
Will do